src/Microsoft.Identity.Core/CoreLoggerBase.cs

```diff
@@ -31,18 +31,21 @@
 using System.Text;
 using System.Threading.Tasks;
 
-namespace Microsoft.IdentityModel.Clients.ActiveDirectory.Internal.Cache
+namespace Microsoft.Identity.Core
 {
-    internal class TokenCachePlugin
+    internal abstract class CoreLoggerBase : ILogger
     {
-        public static void BeforeAccess(TokenCacheNotificationArgs args)
-        {
-            // Default implementation, do nothing
-        }
+        public static ILogger Default = null;
 
-        public static void AfterAccess(TokenCacheNotificationArgs args)
-        {
-            // Default implementation, do nothing
-        }
+        public abstract void Error(string message);
+        public abstract void ErrorPii(string message);
```

Comment:
>Pii [](start = 34, length = 3)

Pii : newer terminology exists and Pii is not longer fully representing gdpr. EUII, Customer Data, etc. IMO, we should consider using a different terminology. I can only think of bad names right now.